src/main/java/com/springboot/connectmate/controllers/InsightController.java

```diff
@@ -21,7 +21,7 @@
 @RequestMapping("/api/insights")
 @Tag(
         name = "Insight REST API",
-        description = "An API that have the CRUD services for insights in the Call Center"
```

Comment:
"An API that has" en vez de "An API that have", pero no es indispensable cambiarlo

Reply:
Va, ¡muchas gracias por la sugerencia Abi! (buena observación) el texto ya está actualizado.